### PR TITLE
Add CRD annotations for service-binding-operator.

### PR DESCRIPTION
**Pull request by pmacik**
This PR:
 * Add annotations to the metadata of the CRD to support https://github.com/redhat-developer/service-binding-operator/pull/183

### CODE REVIEW

File: deploy/crds/postgresql_v1alpha1_database_crd.yaml [hunk 1]
```diff
@@ -2,6 +2,18 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: databases.postgresql.baiju.dev
+  annotations:
+    servicebindingoperator.redhat.io/spec.dbName: 'binding:env:attribute'
```
Comment (otaviof) [inline deploy/crds/postgresql_v1alpha1_database_crd.yaml:6]: @pmacik in general, I would not add any quotes to a string in YAML. Just a minor comment in this PR :-)

Comment (pmacik) [reply deploy/crds/postgresql_v1alpha1_database_crd.yaml:6]: @otaviof Thx for the tip... why not? Does it affect anything in a bad way? :)

Comment (otaviof) [reply deploy/crds/postgresql_v1alpha1_database_crd.yaml:6]: @pmacik nope, it's just cosmetics :-)

**Pull request merged**